dev_environment/setup_linux_dev_env.sh

```diff
@@ -170,6 +182,9 @@ else
   # Start and enable Grafana
   systemctl restart grafana-server
   systemctl enable grafana-server.service
+
+  # Start OTEL collector
+  ./root/otelcol --config=/etc/otelcol/config.yml &
```

Comment:
Modify this to absolute path

Reply:
So can I change it into `./otelcol --config=/etc/otelcol/config.yml &`?

Reply:
Modify path to `/usr/local/bin`